src/thirdparty.ts

```diff
@@ -149,13 +147,12 @@ export function fromEtherWallet(input: string | EtherWalletOptions, password: st
     // derive key/iv using OpenSSL EVP as implemented in CryptoJS
     const evp = evp_kdf(Buffer.from(password), cipher.salt, { keysize: 32, ivsize: 16 })
 
-    const decipher = crypto.createDecipheriv('aes-256-cbc', evp.key, evp.iv)
-    privateKey = runCipherBuffer(decipher, Buffer.from(cipher.ciphertext))
+    const pr = await decrypt(Buffer.from(cipher.ciphertext), evp.key, evp.iv, 'aes-256-cbc')
```

Comment:
What is being guarded against by this `Buffer.from(cipher.ciphertext)`?  Based on typing, this should already be a `Buffer`.

Reply:
cipher.ciphertext is mutable, Buffer.from() could be just a way of copying it to a separate buffer, but not really sure